rails-commenting.rb

```diff
@@ -6,39 +6,39 @@
 
 # FILE: app/controller/blog_posts_controller.rb
 
-# ---1)
+# ---1) This will create the area in your rails application where you will create controller methods
```

Comment:
Good job on this section.  There are a few of your answers that could have been expanded a little further, but overall the concepts of this section are there!